src/org/javarosa/core/reference/ReferenceManager.java

```diff
@@ -49,7 +49,7 @@ private ReferenceManager() {
 	 * @return Singleton accessor to the global
 	 * ReferenceManager.
 	 */
-	public static ReferenceManager _() {
+	public static ReferenceManager instance() {
```

Comment:
Yeay finally :) '_' might not being supported in the future Java release warning was really annoying